hackernews/program.py

```diff
@@ -0,0 +1,59 @@
+"""This project monitors Hacker News for new articles on a specific topic provided via a Slack command
+and posts updates to a Slack channel in real-time."""
```

Comment:
The first line in docstrings should be as short as possible. If you want to add more details, you can add them in additional paragraphs (separated by empty lines) below this first line.

```suggestion
"""Monitor Hacker News for new articles on a specific topic, and post updates to a Slack channel."""
```